tests/stub/disconnects/scripts/return_1.script

```diff
@@ -0,0 +1,17 @@
+!: BOLT 4.3
+!: AUTO HELLO
+
+{*
+    C: RESET
+    S: SUCCESS {}
+*}
+C: RUN "*" "*" "*"
+S: SUCCESS {"fields": ["n.name"]}
+C: PULL {"n": "*"}
+S: SUCCESS {"type": "w"}
```

Comment:
Am I missing something or does this script return nothing?

Reply:
No need to return records for this test. But the script's name is misleading indeed.